pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -589,6 +606,14 @@ impl CustomScan for PdbScan {
             None,
         );
 
+        // Add more detailed partial sort info if applicable
+        if state.custom_state().is_partial_sort {
+            if let Some(sort_field) = &state.custom_state().sort_field {
+                explainer.add_text("Sort Field", sort_field);
+                explainer.add_text("Partial Sort Flag", "True");
+            }
+        }
+
```

Comment:
Is this redundant with the information on lines 652-657?

Reply:
Consolidated them both and did a little improvement.